modules/nodejs.nix

```diff
@@ -0,0 +1,112 @@
+{ config, pkgs, ... }:
+
+let
+
+  typescript-language-server = pkgs.nodePackages.typescript-language-server.override {
+    nativeBuildInputs = [ pkgs.makeWrapper ];
+    postInstall = ''
+      wrapProgram "$out/bin/typescript-language-server" \
+        --suffix PATH : ${pkgs.lib.makeBinPath [ pkgs.nodePackages.typescript ]} \
+        --add-flags "--tsserver-path ${pkgs.nodePackages.typescript}/lib/node_modules/typescript/lib/"
+    '';
+  };
+
+  nodejs = pkgs.nodejs;
+
+  prybar = pkgs.replitPackages.prybar-nodejs;
+
+  dap-node = pkgs.callPackage ../pkgs/dapNode { };
+
+  run-prybar = pkgs.writeShellScriptBin "run-prybar" ''
+    ${prybar}/bin/prybar-nodejs -q --ps1 "''$(printf '\u0001\u001b[33m\u0002\u0001\u001b[00m\u0002 ')" -i ''$1
+  '';
+
+in
+
+{
+  name = "Node.js Tools";
+  version = "1.0";
+
+  packages = [
+    nodejs
+    typescript-language-server
```

Comment:
can't this be implicitly used in `languageServers`? not sure why it needs to be included in `packages` - also plays nicer with deployments